examples/svelte-kitchen-sink/src/stories/__snapshots__/addon-actions.stories.storyshot

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I think this kind of snapshots is not really helping. You should probably implement the rendering function to really render the svelte components into html. 

I see you've added a transformer to the `jest.cofnig`, so probably you should make it something like in the `html/renderTree.js`

Reply:
I took the advice and implemented something similar to the html example. It seems to work.